src/api/public/apidocs-new/paths/request_id.yaml

```
@@ -101,6 +106,11 @@ post:
       schema:
         type: string
       description: Define the new state
+    - in: query
+      name: superseded_by
+      schema:
+        type: string
+      description: Only in combination with the `changestate` command - Supersede the given request with another request by passing its ID in this parameter
```

```suggestion
      description: Required and used only with the combination of the `changestate` command and `newstate=superseded` - Supersede the given request with another request by passing its ID in this parameter
```
I missed that this paremeter is used and required only with the combination of parameters shown above.

---

I have applied the suggestion.